mlflow/java/client/src/test/java/org/mlflow/tracking/MlflowClientTest.java

```diff
@@ -544,7 +544,7 @@ public void checkParamsAndMetrics() {
       Arrays.asList(1L, -5L, 4L, 4L));
 
     List<RunTag> tags = run.getData().getTagsList();
-    Assert.assertEquals(tags.size(), 2);
+    Assert.assertEquals(tags.size(), 1);
```

Comment:
This was recently changed to 2 because we now expect an autogenerated name tag. Back to 1 because that name stays out of the tags :)